src/main/java/com/google/sps/servlets/DisplayServlet.java

```diff
@@ -0,0 +1,51 @@
+package com.google.sps.servlets;
+import com.google.cloud.datastore.Datastore;
+import com.google.cloud.datastore.DatastoreOptions;
+import com.google.cloud.datastore.Entity;
+import com.google.cloud.datastore.Query;
+import com.google.cloud.datastore.FullEntity;
+import com.google.cloud.datastore.QueryResults;
+import com.google.cloud.datastore.StructuredQuery.OrderBy;
+import com.google.cloud.datastore.KeyFactory;
+import com.google.sps.data.Task;
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.swing.text.html.HTML.Tag;
+
+@WebServlet("/DisplayServlet")
```

Comment:
To be consistent with the URL style used by the other servlets, can you change this URL to something like "/display-servlet"?